R/region_to_chunks.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @description Parse a region string into; chromosome, start and end.
 #'
-#' @details To convert region into chunks of chr, start, and end.
+#' @details INTERNAL FUNCTION called by [GAMBLR.utils::calc_mutation_frequency_bin_region], not meant for out-of-package usage.
```

Comment:
I think since helpers are in theory multi-package use cases and can be called outside of GAMBLR we do not necessarily need to refer to a specific package in description.

Reply:
Understood. Should I remove these lines? Or keeping them is fine too?

Reply:
We can focus on these small details after completing separating the child repos and ensuring all the functions continue working - so I think fine to keep as is. You can make an issue for the tracking purposes so this is not forgotten or overlooked later

Reply:
It's not like `#' @details To convert region into chunks of chr, start, and end.` on master anymore. That was before Laura's merged PR.